standard/clause_6_informative_text.adoc

```diff
@@ -481,6 +481,17 @@ To identify a predicate as a variable-to-variable reference predicate, that pred
   <{predicate}> rdfs:range bald:Resource .
 ----
 
+Alternatively, it is permissable to provide an rdfs:range statement targeting another resource and for that resource to declare that it is an rdfs:subClassOf bald:Resource:
+
+----
+ <{attribute}> rdfs:range <{AClass}> .
+
+ <{AClass}> rdfs:subClassOf <https://www.opengis.net/def/binary-array-ld/Resource> .
+----
+
+This standard does not support nested derivation, an implementation is not required to search the inheritance tree of a defined Class to see if any of the classes it inherits from define themselves as rdfs:subClassOf bald:Resource.
```

Comment:
So only direct subclassOf bald:Resource statements are supported to enable V2V refs. Might be worth stating this in the positive as well?

Reply:
l492